R/frequency_encoder.R

```diff
@@ -1,13 +1,18 @@
+
+encode_freq <- function(X) {
+  X
+}
+
 #' frequency_encoder
 #'
 #' This function encodes categorical variables using the frequencies of each category.
 #'
```

Comment:
Maybe add a quick 3 x 3 example of what the frequency encoder does so users can relate to what happen to their data frame?

<img width="483" alt="image" src="https://user-images.githubusercontent.com/54871632/76091725-ba597200-5f83-11ea-8d30-6d4ee9a89e4a.png">

Reply:
I am speculating that this is what the function does.